submissions/ivan-chukhalo/frogger-game/app.js

```diff
@@ -0,0 +1,116 @@
+const AVAILABLE_ROWS_FOR_ENEMIES = [63, 146, 229]; 
+const allEnemies = [];
+
+const WIDTH_OF_GAME_SPRITES = 100;
+const COLUMNS_ON_GAME_FIELD = 5;
+const HEIGH_OF_GAME_FIELD_CELL = 83;
+const PLAYER_BUG_CENTERS_DISTANCE_OF_TOUCHING = 80;
+const Y_OF_GAME_FIELD_ROWS = [63, 146, 229, 312, 395];
```

Comment:
There must be some formula to calculate pixel Y from row number...
Good news is that computers are made to do calculations for us. We need to benefit from this fact to make our lives better.